Backpack/src/main/java/net/skyscanner/backpack/button/internal/BpkButtonBase.kt

```diff
@@ -83,15 +69,7 @@ abstract class BpkButtonBase internal constructor(
     isClickable = isEnabled
 
     this.context.theme.obtainStyledAttributes(attrs, R.styleable.BpkButton, defStyleAttr, 0)
-      ?.use {
-        // TODO: Try to refactor this so using getColorStateList is not necessary.
-        // We are using getColorStateList because we can't set consistent default because of how
-        // BpkButton changes the default colour based on its type.
-        // Furthermore using getColor is problematic because we need to provide a default and if we
-        // make INVALID_RES (-1) the default, it is not possible to se this property to white, because
-        // white is also represented as -1.
-        _buttonTextColor = it.getColorStateList(R.styleable.BpkButton_buttonTextColor)?.defaultColor
```

Comment:
Are you sure this is not required anymore? If I remember correctly theming the button with white text was not working and that's is why we introduced this

Reply:
I remember this as well, but we're using the fallbacks now (which was not possible in the past due to the inheritance). Everything should be fine.